.github/workflows/test.yml

```diff
@@ -183,8 +183,10 @@ jobs:
       run: |
         docker load --input image-master.tar
 
-    - name: Check out code into the Go module directory - from PR branch
+    - name: Check out code into the Go module directory - from Master branch
```

Comment:
the changes in this file shouldn't be in the first commit right?

Reply:
Fixed